File: tests/fixtures/selenium-chaos-suite/src/core/Logger.ts
export class Logger {
  static info(message: string, payload?: any) {
    console.log(`[INFO] ${new Date().toISOString()} - ${message}`, payload || "");
  }

  static warn(message: string, payload?: any) {
    console.log(`[WARN] ${new Date().toISOString()} - ${message}`, payload || "");
  }

  static error(message: string, payload?: any) {
    console.log(`[ERROR] ${new Date().toISOString()} - ${message}`, payload || "");
  }
}

File: tests/fixtures/selenium-chaos-suite-csharp/Utilities/Logger.cs
namespace SeleniumChaosSuite.Utilities;

public static class Logger
{
    public static void Info(string message, object? payload = null)
    {
        Console.WriteLine($"[INFO] {DateTime.UtcNow:O} - {message} {payload ?? string.Empty}");
    }

    public static void Warn(string message, object? payload = null)
    {
        Console.WriteLine($"[WARN] {DateTime.UtcNow:O} - {message} {payload ?? string.Empty}");
    }

    public static void Error(string message, object? payload = null)
    {
        Console.WriteLine($"[ERROR] {DateTime.UtcNow:O} - {message} {payload ?? string.Empty}");
    }
}
